mscore/editstyle.cpp

```diff
@@ -420,6 +424,27 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
       comboFBFont->setCurrentIndex(0);
       connect(comboFBFont, SIGNAL(currentIndexChanged(int)), SLOT(on_comboFBFont_currentIndexChanged(int)));
 
+      // chord symbol init
+      harmonyPlay->setChecked(true);
+
+      voicingSelectWidget->interpretBox->clear();
+      voicingSelectWidget->interpretBox->addItem(tr("Jazz"), int(0)); // two-item combobox for boolean style variant
+      voicingSelectWidget->interpretBox->addItem(tr("Literal"), int(1)); // true = literal
```

Comment:
Why those 2 casts to `int`?

Reply:
I started out with `false` and `true` there and wanted to be clear.

Reply:
but Jazz isn't false ;-)

Reply:
According to @peterhieuvu it is ;-)

Reply:
Ah yeah, Jazz and Literal should probably have been a char/boolean enum instead of `true` or `false` to improve readability